api/src/components/affiliations/controller.ts

```diff
@@ -52,10 +52,9 @@ export const destroy = async (
             });
         }
 
-        //check that the publication is live
-        if (publication.currentStatus !== 'DRAFT') {
+        if (publication.currentStatus === 'LIVE') {
```

Comment:
The previous check was alright. This is now allowing corresponding user to remove affiliations while the publication is LOCKED